recipe/meta.yaml

```diff
@@ -60,6 +58,9 @@ requirements:
     - zlib {{ zlib }}
     - zstd {{ zstd }}
     - xsimd 12.1.1
+    - libbrotlidec {{ libbrotlidec }}
+    - libbrotlienc {{ libbrotlienc }}
+    - libthrift {{ libthrift }}
```

Comment:
I'm a bit curious why these are needed now and weren't before?

Reply:
Yup, introduced in cbc by this commit https://github.com/AnacondaRecipes/aggregate/commit/a5ca748718fed1aed478dd017e7e7e49b2e7cc8f